src/components/layout/GenericLayout/index.tsx

```diff
@@ -10,9 +10,12 @@ export interface Props {
 export const GenericLayout: React.FC<Props> = ({ children, menu, navTools }) => (
   <div>
     <StaticGlobalStyle />
-    <Header menu={menu} tools={navTools} />
-    {children}
-    <Footer />
+    <ThemeProvider>
```

Comment:
Them and layout could be decoupled, and each app decides which ThemeProvider to use.
However, I think for now I think is good your current approach, we only have a single ThemeProvider and it's good not to head to ahead of ourselves

Reply:
:+1: